### DISCUSSION TITLE
Rowy Cloud not seeing my Firebase projects

### DISCUSSION THREAD

**Discussion by dgorshkov** [AntlerVC/firetable#1104] [Support / Q&A] [answered]
1. Was setting up a new project on https://rowy.app
2. "Choose a data source" -> Firebase
3. "Sign in with Google account" -> done
4. "Go to Firebase console and create a project" -> clicked button, made a fresh project, and a fresh Firestore db
5. "Click verify" -> clicked button -> "Looks like you still don't have a Firebase project setup."
6. Waited a bit, tried again -> same result.

**Comment by BeeBombshell** [comment]
Hey @dgorshkov, could you share screenshots or a screen recording of this issue? It'd be helpful to have a screenshot of the console log statements as well.

**Reply by BeeBombshell** [reply]
Hey @dgorshkov, has this issue been resolved for you?

**Reply by dgorshkov** [reply]
@BeeBombshell it is not resolved, unfortunately. Where can I send the screen recording? It looks like I can't just attach it here.

**Reply by BeeBombshell** [reply]
It should generally let you attach a video here, but if that's not working, you could possibly share a drive link or a jam.dev link.

**Reply by dgorshkov** [reply]
@BeeBombshell here ya go! 
https://drive.google.com/file/d/120bE_XoAbY7SqBbVjbhkjDz8q4K_xcrx/view?usp=sharing

**Comment by BeeBombshell** [comment, marked as answer]
@dgorshkov, I believe the account you're logged in on the Firebase is different than the one you're using to create a project on Rowy (different profile picture). You could simply switch the account on Firebase to get it to work perfectly. Hope this helps!

**Reply by dgorshkov** [reply]
Huh. Looks like the "Open Firebase Console" button was taking me to the wrong place.

**Reply by BeeBombshell** [reply]
The other account might have been the default one, which is the reason why this was happening. Hope it is fixed now.

**Reply by dgorshkov** [reply]
No, it's not fixed, the button is taking me to the wrong account.

**Reply by BeeBombshell** [reply]
@dgorshkov, when you get redirected to the Firebase console, you can simply click on your picture and select the other account. 
> Refer to the screenshot below
<img width="1512" alt="Screenshot 2023-02-22 at 4 53 03 PM" src="https://user-images.githubusercontent.com/53828745/220606935-e0831bf5-46f6-426c-a88a-1719d7a1cad3.png">
